hack/istio-bookinfo-openshift.sh

```diff
@@ -8,7 +10,7 @@ if [ "$?" != 0 ]; then
 fi
 
 # find istio that we installed from the install script
-INSTALL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )/_output/istio"
+INSTALL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )/_output/istio"
```

Comment:
are you sure about this?  The script is in the hack directory. Go up to the parent and there is our build's "_output" directory. Going up two parent directories seems incorrect. Am I missing something?

What we want is to reuse the _output directory that is used by the Makefile. This directory: `<root>/src/github.com/swift-sunshine/swscore/_output`

Reply:
@jmazzitelli that was my ask too.